src/avro2json.c

```diff
@@ -714,6 +722,20 @@ static int avro_value_to_csv(FILE *dest, const avro_value_t *value,
     int64_t val;
     CHECKED_EV(avro_value_get_long(value, &val));
 
+    if (transformation != NULL) {
+      if(strcmp(transformation, TS_SECS_TRANSFORM) == 0) {
```

Comment:
I would avoid calling `strcmp` on every record. Can you parse the transformations into some enum in advance?

Reply:
> I would avoid calling `strcmp` on every record. Can you parse the transformations into some enum in advance?

Great catch! Done.